test/smoke/dapi.js

```diff
@@ -1,10 +1,45 @@
+const DAPIClient = require('@dashevo/dapi-client');
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const networkConfig = getNetworkConfig();
+
+const fetch = require('node-fetch');
+
 describe('DAPI', () => {
-  // getBlockHash height:1
-  it('should respond data from chain');
+  networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
+    describe(`DAPI ${nodeName}`, () => {
+      const options = {
+        debug: false,
+        verbose: false,
+        errors: false,
+        warnings: false,
+        seeds: [{ ip: networkConfig.inventory._meta.hostvars[nodeName].public_ip }],
+      };
+      const dapi = new DAPIClient(options, 3000);
+      // getBlockHash height:1
+      it('should respond data from chain', async () => {
+        const blockHash = await dapi.getBlockHash(1);
+        // const response = await utils.requestWrapper(`http://${networkConfig.inventory._meta.hostvars[nodeName].public_ip}:3001/insight-api-dash/block-index/1`);
+        const resp = await fetch(`http://${networkConfig.inventory._meta.hostvars[nodeName].public_ip}:3001/insight-api-dash/block-index/1`)
+          .then(res => res.text());
+        const blockHashInsight = JSON.parse(resp).blockHash;
+        expect(blockHash).to.be.an('string');
+        expect(blockHash).to.be.equal(blockHashInsight);
+      });
 
-  // getBestBlockHeight
-  it('should respond data from insight');
+      // getBestBlockHeight
+      it('should respond data from insight', async () => {
+        const blockHeight = await dapi.getBestBlockHeight(1);
+        expect(blockHeight).to.be.an('number');
+        expect(blockHeight % 1).to.be.equal(0);
+      });
 
-  // fetchDapObjects any dapiId and type
-  it('should respond data from drive');
+      // fetchDapObjects any dapiId and type
+      it('should respond data from drive', async () => {
+        const balance = await dapi.getBalance(`${networkConfig.variables.faucet_address}`);
```

Comment:
It supposes to make a request to drive?

Reply:
You can call `getDapContract` or `getUserDapSpace`.

Reply:
there are fetchDapContract &  fetchDapObjects instead of getDapContract/getUserDapSpace but they need valid dapId in params. so, to fetch by dapId firstly we need create this object. I guess it's not a sanity test and should be part of integration scenarios.

Reply:
No worries, those methods respond you that they don't have these objects/contract but that's enough for us to make sure that services are working and connected.

Reply:
I see that fetchDapObjects  has been added 5 days ago https://github.com/dashevo/dapi-client/blob/master/src/index.js . when I call it I get 
Error: DAPI RPC error: fetchDapContract: Method not found
      at Object.request (node_modules/@dashevo/dapi-client/src/RPCClient.js:30:11)

do we need to update dapi instances?

Reply:
Yep, @antohou please push new changes.

Reply:
dapi image has been updated. now dashdrive lib should be updated